build.gradle

```diff
@@ -1,12 +1,14 @@
 buildscript {
-    ext.kotlin_version = '1.3.20'
-    ext.ktor_version = '1.1.2'
+    ext.kotlin_version = '1.3.61'
+    ext.ktor_version = '1.2.6'
+    ext.serialization_version = '0.14.0'
```

Comment:
Some extension method were using those apis but after other libs update above one needed explicit version as it was failing the build.